ext/hash/hash.stub.php

```diff
@@ -2,37 +2,37 @@
 
 /** @generate-function-entries */
 
-function hash(string $algo, string $data, bool $raw_output = false): string|false {}
+function hash(string $algorithm, string $data, bool $binary_output = false): string|false {}
```

Comment:
I could also imagine a `$binary_format` parameter name, because `output` suggests that the result is printed, rather than returned.

Reply:
I don't like the `$binary_output` name, but having a hard time coming up with a better one. TBH I slightly prefer the current `$raw_output`.

Another possibility would be `$no_hex`. Negative names are not good, but I think this may be a more accurate description of what the parameter really controls.

Reply:
The doc description of this parameter is "When set to TRUE, outputs raw binary data. FALSE outputs lowercase hexits."

(... hexits?)

So, whatever the parameter is should have "binary" in the name.  `$use_binary`, `$return_binary`, maybe just `$binary`, but it should be in there somewhere.

```php
hash(
  algorithm = 'md5',
  return_binary = true,
);
```

Reply:
I just noticed that there are hash_algos() and hash_hmac_algos() functions, which suggests the parameter should maybe stay at `$algo` for consistency. Don't feel strongly about that though.

As to the binary parameter, just `$binary` would work for me (though isn't great -- there doesn't seem to be a great name, because the meaning of this parameter should have been inverted to start with.)

Reply:
Ah, I was just about to ask you how you feel about `$binary`. :) It's shorter than the current name, but still much more expressive than `$raw_output` IMO.

Reply:
Speaking about the `$algo` name: ext/openssl uses `$algorithm` everywhere (khmm, it's not a coincidence though... ^^), so we should change `$cipher_algo` and `$digest_algo` there as well